content/reference/rest-api/overview/authentication.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Authentication'
+weight: 10
+
+menu:
+  main:
+    identifier: 'rest-api-authentication'
+    parent: 'rest-api-overview'
+---
+
+Every request your application sends to Cawemo's public API requires the use of HTTP Basic Authentication.
+This means that in order to make a successful request, you must pass a valid user id and api key combination as an authorization header for each request.
+The user id and api key combination can be retrieved from the settings page of your organization.
+
+**TODO: Add image**
+
+When building a request using Basic Authentication make sure you always add the `Authentication: Basic` HTTP header with the Base64-encoded credentials.
+
+In the following cURL request example, you would replace \<user_id> and \<api_key> with your credentials before sending the request:
+
+```
+curl -v -u <user_id>:<api_key> "https://cawemo.com/api/v1/catalogs"
```

Comment:
We haven't agreed on a url where we want to make the api available - ideally it would be something simple like `https://api.cawemo.com` but that might be too complicated to achieve for on-premise installations in the first iteration.

Unfortunately, our internal api is already using `/api` as sub-path so putting the public api below `/api/v1` like here could be confusing because it wouldn't reflect the different concerns. Do you maybe have a better idea where to make the public api accessible?

Reply:
See https://github.com/camunda/camunda-docs-cawemo/pull/118#discussion_r685935385 - I opened https://github.com/camunda/cawemo/issues/8848 for moving our internal API to a different location